config/routes.rb

```diff
@@ -2,6 +2,12 @@
   devise_for :users
 
   namespace :editorial do
+    get '/news' => 'news#index', as: :news_articles
```

Comment:
I think [you can do](http://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing):

````
resources :news, only: [:create, :index, :new], path: 'news_articles'
````